gudrun_classes/enums.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+
+class Scales(Enum):
+    Q = 1
+    D_SPACE = 2
```

Comment:
We would commonly refer to this as
```suggestion
    D_SPACING = 2
```